examples/omega2/omega2.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+
+import subprocess
+
+
+class Omega2(object):
+    def __init__(self):
+        self.version = self.get_omega_version()
+        self.led_path = "/sys/class/leds/%s:amber:system/" % self.version
+
+        # https://docs.onion.io/omega2-docs/gpio-python-module.html
+        self._gpio_method = ['shell', 'python'][0]
+        # https://docs.onion.io/omega2-docs/using-gpios.html#fast-gpio
+        self._gpio_tool = ['gpioctl', 'fast-gpio'][0]
+
+        if self._gpio_method == 'python':
+            # import onionGpio
+            pass  # TODO
+
+    @staticmethod
+    def get_omega_version():
+        """Returns 'omega2' for Omega2 and 'omega2p' for Omega2+"""
+        return str(subprocess.check_output(
+            ["uci", "get", "system.@led[0].sysfs"])).split(":")[0]
+
+    def led_control(self, state, delay_on=None, delay_off=None,
```

Comment:
This is an Omega2 onboard LED https://docs.onion.io/omega2-docs/the-omega-led.html I thing it is strongly device-stecific